### DISCUSSION TITLE
Ability to make a custom order of Power Modes

### DISCUSSION THREAD

**Discussion by Malkozaine** [seerge/g-helper#3340] [Ideas]
I have the Ally X and your program is amazing.  I was wondering if it would be possible to add the ability to customize the order of Power Modes.  I made a small handful and would love to order them by their power levels.

**Comment by seerge** [comment]
@Malkozaine hello,

Modes are already ordered by their default power levels (i.e. Silent -> Balanced > Turbo) :)
